modules/platform/video/pom.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.ecm.platform</groupId>
+    <artifactId>nuxeo-platform-parent</artifactId>
+    <version>11.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nuxeo-platform-video-parent</artifactId>
+  <name>Nuxeo Platform Video</name>
+  <packaging>pom</packaging>
+  <description>Nuxeo Platform Video provides video management to Nuxeo Web Platform and RCP.</description>
+
+  <modules>
+    <module>nuxeo-platform-video</module>
+    <module>nuxeo-platform-video-rest</module>
+  </modules>
+
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <configuration>
+              <archive>
+                <manifestFile>src/test/resources/META-INF/MANIFEST.MF</manifestFile>
+              </archive>
+            </configuration>
+            <goals>
+              <goal>test-jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Not needed, this is useful to publish a test jar, e.g. `my-module-11.5-SNAPSHOT-tests.jar`, which only makes sense if the module has some interesting things to expose in its `src/test` classpath (typically test features implementing `RunnerFeature`), which is not the case here.